### PR TITLE
Bug fixes, notation, and status components.

### PR DESCRIPTION
**Pull request by quinn-madson**
Added chess-score-pad component to display notation. Added chess-game-controls component to start playing. Bug fixes for castling and pawn promotion.

### CODE REVIEW

File: src/chess/chess-arbiter.js [hunk 1]
```diff
@@ -27,28 +32,57 @@ AFRAME.registerSystem("chess-arbiter", {
...
-	},
+  addEventListeners() {
+    this.el.sceneEl.addEventListener("chess:playAs", this.playAsEvent);
+    this.el.sceneEl.addEventListener("chess:copyPGN", this.copyPGN);
+    this.el.sceneEl.addEventListener("chess:copyFEN", this.copyFEN);
+  },
+
+  removeEventListeners() {
+    this.el.sceneEl.removeEventListener("chess:playAs", this.playAsEvent);
+  },
```
Comment (wmurphyrd) [inline src/chess/chess-arbiter.js]: Should the add & remove methods do all the same listeners?

File: src/chess/chess-arbiter.js [hunk 2]
```diff
@@ -59,6 +93,9 @@ AFRAME.registerSystem("chess-arbiter", {
       case "b":
         this.playAs("black", id, profile);
         break;
+      case "fen":
+        this.resetNetworkedGame(fen);
+        break;
```
Comment (wmurphyrd) [inline src/chess/chess-arbiter.js:104]: aww yes here we go

**Review by wmurphyrd — COMMENTED** [review]
Just a question on the listener de-registration. I didn't play through to test the advanced moves; figured we'd do that soon enough :)

File: src/chess/chess-arbiter.js (hunk 1 above)
Comment (quinn-madson) [reply src/chess/chess-arbiter.js]: Yeah, oversight on my part. I'll correct it and update the PR.

Comment (quinn-madson) [reply src/chess/chess-arbiter.js]: Fixed in commit 3a62277.

**Pull request merged**